.ci/publish-releasenotes-outside.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z $1 ]]; then
+  echo "latest version is not set"
```

Comment:
remove `latest`
put also `echo` example of usage.

Reply:
Changes Done